client/http/http_sender_test.go

```diff
@@ -15,19 +15,22 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
-package client
+package http
```

Comment:
Once we unexport `{rpc,http}sender.New()`, these packages will no longer have any exported names; they will only be imported for their registration side effects. In that case it doesn't really matter what the package is named since it will only be imported as `_`, so I think `import _ "github.com/cockroachdb/cockroach/client/rpc"` looks a little nicer than `rpcsender`.